mantl/mantl-dns/spec.yml

```diff
@@ -29,7 +29,7 @@ scripts:
 
   after-install: |
     # update the installed config search list with resolv.conf values
-    search=$(grep ^search /etc/resolv.conf.masq)
+    search=$(grep ^search /etc/resolv.conf.masq | sed -e "s/consul\s//")
```

Comment:
does /etc/resolv.conf.masq read by dns-masq? or it just backup copy of resolv.conf?